com.ranfdev.Geopard.json

```diff
@@ -0,0 +1,41 @@
+{
+    "app-id" : "com.ranfdev.Geopard",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "42",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "geopard",
+    "finish-args" : [
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=network"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "env": {
+            "CARGO_HOME": "/run/build/Geopard/cargo",
+            "RUST_LOG": "info"
+        }    
+    },
+    "modules" : [
+        {
+            "name" : "Geopard",
+            "buildsystem" : "meson",
+            "config-opts": [
+                "-Dbuildtype=release",
+                "-Doffline=true"
+            ],
+            "sources" : [
+                "generated-sources.json",
+                {
+                    "type" : "archive",
+                    "path": "/home/lollo/Importante/Progetti/Geopard/_build/meson-dist/geopard-1.0.0.tar.xz",
```

Comment:
and how to you think it is find the source?

Reply:
Thank you for your patience. I thought I had updated that line...